### PR TITLE
Added new Contributor Ritwik Mehta

### PR DESCRIPTION
**Pull request by Ritwikmehta28**
This commit adds new CodeforCause Contributor -Ritwik Mehta.

### CODE REVIEW

File: src/data/contributors/RitwikMehta.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Ritwik Mehta",
+  "skills": ["C++", "Flutter/Dart", "FullStack"],
+  "cause": "I code to build. To be able to teach one day.",
+  "image": "Avatar3",
+  "github": "https://github.com/Ritwikmehta28",
+  "twitter": "https://www.geekyhermit.com/",
```
Comment (GangaChatrvedi) [inline src/data/contributors/RitwikMehta.json:7]: You need to specify link of your twitter account here if you wish to add or you can keep this blank.

**Comment by Ritwikmehta28** [comment]
Done as requested Mam.

File: src/data/contributors/RitwikMehta.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Ritwik Mehta",
+  "skills": ["C++", "Flutter/Dart", "FullStack"],
+  "cause": "I code to build. To be able to teach one day.",
```
Comment (GangaChatrvedi) [inline src/data/contributors/RitwikMehta.json:4]: "I code for" is already there in the string so you only have to write the cause you code for.

**Comment by Ritwikmehta28** [comment]
Made the changes in the string.

**Pull request merged**